File: Assets/GraphQL/Model/Trips/Trip.cs
using GraphQL.Helpers;

namespace GraphQL.Model {
	public class Trip {
		[GqlSelection("tripID")] public int TripID { get; private set; }
		[GqlSelection("userID")] public int UserID { get; private set; }
		[GqlSelection("name")] public string Name { get; private set; }
		[GqlSelection("guideID")] public int GuideID { get; private set; }
		[GqlSelection("consultantID")] public int ConsultantID { get; private set; }
		[GqlSelection("dateTimeID")] public int DateTimeID { get; private set; }
		[GqlSelection("transportRentID")] public int TransportRentID { get; private set; }
	}
}

File: Server/src/schema/Trips/trip.ts
import { Field, InputType, Int, ObjectType } from "type-graphql";
import { BaseEntity, Column, Entity, PrimaryGeneratedColumn, Timestamp } from "typeorm";

@Entity()
@ObjectType()
export default class Trip extends BaseEntity {
    @Field(() => Int!)
    @PrimaryGeneratedColumn()
    tripID!: number;

    @Field(() => Int!)
    @Column()
    userID!: number;

    @Field(() => String!)
    @Column()
    name!: string;

    @Field(() => Int!)
    @Column()
    guideID!: number;

    @Field(() => Int!)
    @Column()
    consultantID!: number;

    @Field(() => Int!)
    @Column()
    dateTimeID!: number;

    @Field(() => Int!)
    @Column()
    transportRentID!: number;

    constructor() {
        super();

        this.tripID = 0;
        this.userID = 0;
        this.name = "None";
        this.guideID = 0;
        this.consultantID = 0;
        this.dateTimeID = 0;
        this.transportRentID = 0;
    }
}